src/main/target/common_defaults_post.h

```diff
@@ -58,6 +58,7 @@
 #ifndef RX_SPI_DEFAULT_PROTOCOL
 #define RX_SPI_DEFAULT_PROTOCOL 0
 #endif
+
 #ifndef SERIALRX_PROVIDER
```

Comment:
Please remove these lines - prefer to have this logic in rx.c as it is closer to where it is used.

Reply:
Yes they are there for 5 years. Preventing the macro in rx.c